torch_em/transform/generic.py

```diff
@@ -129,11 +134,18 @@ def _get_preprocess_shape(self, oldh, oldw):
         newh = int(newh + 0.5)
         return (newh, neww)
 
+    def _convert_transformed_inputs_to_original_shape(self, resized_inputs):
+        # First step is to remove the padded region
+        inputs = resized_inputs[tuple(self.pre_pad_shape)]
```

Comment:
I would add a check here that `pre_pad_shape` exists. and raise an error if it doesn't exists that informs that this is only valid to be used after `__call__` has run.
You can check this via
```python
if not hasattr(self, "pre_pad_shape"):
   ...
```

Reply:
Ahha makes sense. Thanks for the feedback!